package.json

```diff
@@ -102,6 +102,7 @@
     "**/jest-jasmine2": "npm:@amoo-miki/jest-jasmine2@26.4.2-rc.1",
     "**/jsonpointer": "^5.0.0",
     "**/json-schema": "^0.4.0",
+    "**/jszip": "^3.8.0",
```

Comment:
This shouldn't be needed.

Reply:
Yeah it is true. If we bootstrap, we will get an update.

Reply:
I removed the resolution and did a clean bootstrap as recommended.